packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -220,11 +220,8 @@ function getStylesDeclarations( blockStyles = {} ) {
 
 	// The goal is to move everything to server side generated engine styles
 	// This is temporary as we absorb more and more styles into the engine.
-	const extraRules = getCSSRules( blockStyles, { selector: 'self' } );
```

Comment:
Just making sure I understand — the idea here is that we treat the absence of specifying a selector as "give me the values that are allowed to be used as inline styles", so we don't need to pass it `'self'`? If so, that makes sense to me 👍

Reply:
>  the idea here is that we treat the absence of specifying a selector as "give me the values that are allowed to be used as inline styles", so we don't need to pass it 'self'?

Yeah, I though 'self' didn't really communicate much so, in the absence of `options.selector` we'd just return the concatenated style values.